libkbfs/bare_root_metadata_v3.go

```diff
@@ -88,6 +88,16 @@ type BareRootMetadataV3 struct {
 	// of writing to the given folder.
 	FinalizedInfo *tlf.HandleExtension `codec:"fi,omitempty"`
 
+	// The sequence number of the global Keybase Merkle tree at the
+	// time this update was created (from the writer's perspective).
+	// This field was added to V3 after it was live for a while, and
+	// older clients that don't know about this field yet might copy
+	// it into new updates via the unknown fields copier. Which means
+	// new MD updates might end up referring to older Merkle roots.
+	// That's ok since this is just a hint anyway, and shouldn't be
```

Comment:
Yes to all questions.  We can't just rely on this hint since the clients could be lying about it.  We can only use it as a starting point for finding the right place in the merkle trees for comparing the update to the team membership around the time of the update.  Fetching those Merkle records and verifying them is where the expense will be, I think.